kitsune/wiki/tests/test_views.py

```diff
@@ -177,6 +177,22 @@ def test_custom_event_while_fallback_locale(self):
         assert '"Fallback Locale"' in doc('body').attr('data-ga-push')
         assert '"Not Localized"' not in doc('body').attr('data-ga-push')
 
+    def test_en_in_accept_language_header(self):
```

Comment:
I dont see any ``accpeted_language_header`` is passed through the client for fallback. can you please check other tests where the header is passed and tested?